docs/application/distribute/how-to-distribute/overview.md

```diff
@@ -2,12 +2,12 @@
 
 ## Overview of Application Registration
 
-- The following is the information required for a seller to register an application. After entering information required in the Binary, Sales, and Display tabs, you can submit your application. 
+- The following is the information required for a seller to register an application. After entering information required in the Binary, Sales, and Display tabs, you can submit your application.
 
 | **Tab**                                  | **Description**                          |
 | ---------------------------------------- | ---------------------------------------- |
-| [**Binary**](../how-distribute/binary) | - [Application Title (Default Language)](../how-distribute/binary.md#ApplicationTitle) <br/>- [Binary File and Supported Device](../how-distribute/binary.md#BinaryFile) <br/>- [Expansion File Upload](../how-distribute/binary.md#Expansion) <br/>- [Validation Information](../how-distribute/binary.md#Validation) |
-| [**Sales**](../how-distribute/sales.md) | - [Country & Price](../how-distribute/sales.md#Country) <br/>- [Availability Date](../how-distribute/sales.md#Availability) <br/>- [Category](../how-distribute/sales.md#Category) <br/>- [Age Restriction](../how-distribute/sales.md#Restriction) |
-| [**Display**](../how-distribute/display.md) |- [Application Detail](../how-distribute/display.md#ApplicationDetails) <br/>- [Icon Image](../how-distribute/display.md#IconImage) <br/>- [Screenshots](../how-distribute/display.md#Screenshots) <br/>- [YouTube URL](../how-distribute/display.md#YouTubeURL) <br/>- [Banner](../how-distribute/display.md#Banner) <br/>- [Support](../how-distribute/display.md#Support) |
-| [**Item**](../how-distribute/item.md) | - [In App Purchase Type & Item Group ID](../how-distribute/item.md) |
+| [**Binary**](../how-to-distribute/binary.md) | - [Application Title (Default Language)](../how-to-distribute/binary.md#application-t) <br>- [Binary File and Supported Device](../how-to-distribute/binary.md#binary-file) <br>- [Expansion File Upload](../how-to-distribute/binary.md#expansion) <br>- [Validation Information](../how-to-distribute/binary.md#validation) |
+| [**Sales**](../how-to-distribute/sales.md) | - [Country & Price](../how-to-distribute/sales.md#country-price) <br>- [Availability Date](../how-to-distribute/sales.md#availability-date) <br>- [Category](../how-to-distribute/sales.md#category) <br>- [Age Restriction](../how-to-distribute/sales.md#age-restriction) |
```

Comment:
It looks like sales.md files doesn't have `<a name="country-price"></a>`,  `<a name="availability-date"></a>` or `<a name="category"></a>`

Reply:
"## Availability Date" is the 2nd title. When markdown file is built, boost convert title to HTML tag with "id" property.
So...we dont need to add <a name="xxx"><a> when title and id are same.